### PR TITLE
Update Copyrights in JetPack to 2021 - 11700

### PR DESCRIPTION
**Pull request by CalvinSuma**
Updated the copyright date from x-2020 to x-2021

### CODE REVIEW

File: src/mgmt/director.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# (c) 2014-2020 Dell
+# (c) 2014-2021 Dell
```
Comment (heydog15) [inline src/mgmt/director.cfg:1]: Why does this file just say Dell and all the others have Dell Inc. or its subsidiaries?

File: src/mgmt/osp-sah.ks [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# (c) 2014-2020 Dell
+# (c) 2014-2021 Dell
```
Comment (heydog15) [inline src/mgmt/osp-sah.ks:1]: Why does this file just say Dell and all the others have Dell Inc. or its subsidiaries?

Comment (CalvinSuma) [reply src/mgmt/osp-sah.ks:1]: That is what it said from the start. There were multiple version did you want them all standardized?

**Comment by heydog15** [comment]
I don't know.  Geal, shouldn't they all be the same?  These are the only 2 out of 150

**Comment by CalvinSuma** [comment]
> I don't know. Geal, shouldn't they all be the same? These are the only 2 out of 150

**Comment by CalvinSuma** [comment]
@heydog15 wrong button there should be a few others that have slightly different messages

**Comment by gaelrehault** [comment]
yes, header format should be the same on all files. possible we missed some last year or that some were added incorrectly during the year.

**Comment by heydog15** [comment]
Gael/David, should they all be the same?  I would think so.

**Comment by heydog15** [comment]
Sorry, thanks

**Comment by heydog15** [comment]
Looks good!!

**Pull request merged**